src/types/zodJira.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+
```

Comment:
Consideration of this file directly related to the using/not using FlawModel for non-OSIDB stuff.